### ISSUE TITLE
dtplyr release?

### ISSUE THREAD

**Issue by hadley** [tidyverse/dtplyr#407]
Any concern with me doing a dtplyr release? (It's broken by dplyr 1.1.0)

Should this be a patch version or a minor version?

**Comment by hadley** [comment]
If someone could implement `.by` soon (maybe in the next week?) I think we could hold off; otherwise I think it's fine to include that in the next minor release.

**Comment by markfairbanks** [comment]
> If someone could implement `.by` soon (maybe in the next week?) I think we could hold off;

Yeah that should be doable, I'll take a look.

**Comment by hadley** [comment]
@markfairbanks also would mind giving me a list of what you think the biggest new features are? (i.e. what are the things that I should highlight in the blog post?)

**Comment by markfairbanks** [comment]
Yep, will do. Also FYI I took an initial pass at `.by` and I'll have a PR by Friday.

**Comment by hadley** [comment]
Awesome, thanks!

**Comment by markfairbanks** [comment]
Here's the list. I grabbed this from the reorganized NEWS from #410. Not sure if all of these should be in the blog post but they're the most notable.


## New features

* New translations:

  * `add_count()`
  * `unite()`

* `.by`/`by` has been implemented for `mutate()`, `summarise()`, `filter()`,
  and the `slice()` family (#399)

* `min_rank()`, `dense_rank()`, `percent_rank()`, & `cume_dist()` are now mapped
  to their `data.table` equivalents (#396)
  
* `pick()` is now translated (#341)

* `across()` output can now be used as a data frame (#341)

## Performance improvements

* `select()` now drops columns by reference when possible for improved performance (#367)

* `arrange()` now utilizes `setorder()` when possible for improved performance (#364)

* `slice()` uses an intermediate variable to reduce computation time of row selection (#377)

## Minor improvements and bug fixes

* `dplyr` and `tidyr` verbs no longer dispatch to `dtplyr` translations when used
  directly on `data.table` objects. `lazy_dt()` must now explicitly be called by
  the user. (#312)

**Comment by hadley** [comment]
Thanks!

I'd suggest moving the last point to a new "breaking changes" heading at the top of the news for this release. I don't think it will affect many people, but it's good to advertise prominently.

**Comment by markfairbanks** [comment]
Sounds good

**Comment by hadley** [comment]
Closing this issue and about to open an official release issue.
